File: bonus/Colleen.py
def cc():
	return None
#lol
str = "def cc():%c%creturn None%c#lol%cstr = %c%s%c"
str2 = "str2 = %c%s%c%cprint (str %c (10, 9, 10, 10, 34, str, 34))%cprint (str2 %c (34, str2, 34, 10, 37, 10, 37))"
print (str % (10, 9, 10, 10, 34, str, 34))
print (str2 % (34, str2, 34, 10, 37, 10, 37))

File: Colleen.c
#include <stdio.h>
/*
	cc
*/
void lol(){}
int main()
{
	lol();
	char *program = "#include <stdio.h>%c/*%c%ccc%c*/%cvoid lol(){}%cint main()%c{%c%clol();%c%cchar *program = %c%s%c;%c%cprintf(program, 10, 10, 9, 10, 10, 10, 10, 10, 9, 10, 9, 34, program, 34, 10, 9, 10, 9, 10, 9, 9, 10, 9, 10, 9, 10, 10);%c%c/*%c%c%cre cc%c%c*/%c%creturn 0;%c}%c";
	printf(program, 10, 10, 9, 10, 10, 10, 10, 10, 9, 10, 9, 34, program, 34, 10, 9, 10, 9, 10, 9, 9, 10, 9, 10, 9, 10, 10);
	/*
		re cc
	*/
	return 0;
}
